### ISSUE TITLE
cli - Do not create cluster when  ```--k8s-version list``` option is passed.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#12383]
Rancher cli version - rancher-v1.0.0-alpha11

We should not create a cluster when  ```--k8s-version list``` option is passed to it.
This should only list all the supported K8s version.

```
./rancher cluster create --description mytest1 --disable-docker-version --k8s-version list --network-provider flannel mytestcanal
Available Kubernetes versions:
v1.9.4-rancher1-1
v1.9.5-rancher1-1
v1.8.9-rancher1-1
Successfully created cluster mytestcanal
```

**Comment by deniseschannon** [comment]
Available with v1.0.0-alpha12

**Comment by sangeethah** [comment]
Tested with v1.0.0-alpha12

Cluster does not get created when ```--k8s-version list``` option is passed to cluster create command.
